contract/system/param.go

```diff
@@ -86,51 +134,45 @@ func updateParam(s dataSetter, id string, value *big.Int) (error) {
 	return nil
 }
 
-// save the new value for the param, to be active on the next block
-func (p parameters) setNextParam(proposalID string, value *big.Int) {
-	p[proposalID + "next"] = value
-}
-
 // if a system param was changed, apply or discard its new value
 func CommitParams(apply bool) {
 	for i := sysParamIndex(0); i < sysParamMax; i++ {
 		id := i.ID()
 		// check if the param has a new value
-		if systemParams[id + "next"] != nil {
+		if param := systemParams.getNextParam(id); param != nil {
 			if apply {
 				// set the new value for the current block
-				systemParams[id] = systemParams[id + "next"]
+				systemParams.setParam(id, param)
 			}
 			// delete the new value
-			systemParams[id + "next"] = nil
+			systemParams.delNextParam(id)
 		}
 	}
 }
 
 // get the param value for the next block
 func GetNextParam(proposalID string) *big.Int {
 	// check the value for the next block
-	if val, ok := systemParams[proposalID + "next"]; ok {
+	if val := systemParams.getNextParam(proposalID); val != nil {
 		return val
 	}
 	// check the value for the current block
-	if val, ok := systemParams[proposalID]; ok {
+	if val := systemParams.getParam(proposalID); val != nil {
 		return val
 	}
 	// default value
-	return DefaultParams[proposalID]
+	return DefaultParams.getParam(proposalID)
 }
 
 // get the param value for the current block
 func GetParam(proposalID string) *big.Int {
-	if val, ok := systemParams[proposalID]; ok {
+	if val := systemParams.getParam(proposalID); val != nil {
 		return val
 	}
-	return DefaultParams[proposalID]
+	return DefaultParams.getParam(proposalID)
 }
 
 // these 4 functions are reading the param value for the current block
-
 func GetStakingMinimum() *big.Int {
```

Comment:
This comment is for the next 4 function, so it should have an empty line

Reply:
Is these our official convention? all code can be changed to using these convension.

Reply:
personally, i prefer these convension to division line
<img width="893" alt="image" src="https://github.com/aergoio/aergo/assets/28998570/b77a9813-52d1-4aea-898e-8bd0dd1d9c9d">

Reply:
loogs good